src/lib/include/roc/address.h

```diff
@@ -126,6 +139,18 @@ ROC_API const char* roc_address_ip(const roc_address* address, char* buf, size_t
  */
 ROC_API int roc_address_port(const roc_address* address);
 
+/** Get multicast interface address.
+ *
+ * @b Parameters
+ *  - @p address should point to a properly initialized address struct
+ *
+ * @b Returns
+ *  - returns zero-terminated string representation of the multicast address
+ *  - returns NULL if the argument is invalid
+ *  - returns NULL if the multicast address was not set
+ */
+ROC_API const char* roc_address_miface(const roc_address* address);
```

Comment:
This API assumes that roc_address stores the the raw string somewhere. This is true now, but may change in future (e.g. we may want to store a parsed address or a some interface handle instead). So I suggest to remove this assumption from the API and instead copy the miface to the user-provided buffer, like we do it for IP address:

https://github.com/roc-project/roc/blob/a29d8b5edf4f29922c506f5f27fe12dbd752c171/src/lib/include/roc/address.h#L129

Reply:
Fixed